docs/guide/overview.asciidoc

```diff
@@ -38,6 +36,69 @@ This data is buffered for a short period and sent on to APM Server.
 It listens on port 8200 by default and receives data from agents through a JSON HTTP API.
 Then it creates documents from that data and stores them in Elasticsearch.
 
-To visualize the data after it's sent to Elasticsearch,
-you can use the the dedicated APM UI bundled in X-Pack,
-or the pre-built open source Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
\ No newline at end of file
+*Elasticsearch* is a highly scalable open-source full-text search and analytics engine.
+It allows you to store, search, and analyze big volumes of data quickly and in near real time.
+Elasticsearch is used to store APM performance metrics and make use of its aggregations. 
+
+*Kibana* is an open source analytics and visualization platform designed to work with Elasticsearch.
+You use Kibana to search, view, and interact with data stored in Elasticsearch.
+You an use Kibana to visualize APM data by utilizing the dedicated APM UI bundled in Basic license,
+or the pre-built, open source,
+Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
+
+[[concepts]]
+[float]
+=== APM Concepts
+
+[float]
+==== Events
+APM agents capture different types of information from within their instrumented applications, known as events.
+Events can be <<errors,`Errors`>>, <<transaction-spans,`Spans`>>, or <<transactions,`Transactions`>>.
+These events are then streamed to a single endpoint in the APM Server which sorts and processes the events. 
+
+*<<errors,`Errors`>>* contain information about the original exception that occurred,
+or about a log created when the exception occurred.
+
+*<<transaction-spans,`Spans`>>* contain information about a specific code path that has been executed.
+They measure from the start to end of _something_,
+and they can have a parent/child relationship with other spans. 
+
+*<<transactions,`Transactions`>>* are a special kind of span that have metadata associated with them.
```

Comment:
Comparing spans to transactions in this way is just my preferred way of explaining it. But we should probably verify with @makwarth if this also falls in line with the official message we want to send 😅 But I personally like it 👍

Reply:
I'm good with it

Reply:
Just a last minute note: I would probably say "have extra metadata" instead of just "have metadata" as spans also have metadata. But it's not a big issue. Just thought I'd mention it if you think it's a good addition 😃